wandb/sdk/wandb_login.py

```diff
@@ -36,7 +36,9 @@ def _handle_host_wandb_setting(host: Optional[str], cloud: bool = False) -> None
         _api.set_setting("base_url", host, globally=True, persist=True)
 
 
-def login(anonymous=None, key=None, relogin=None, host=None, force=None):
+def login(
+    anonymous=None, key=None, relogin=None, host=None, force=None, login_timeout=None
```

Comment:
This should just be called "timeout" here.

To map this to login_timeout do something like:
wandb_settings.py:
```
    def _apply_login(
        self, args: Dict[str, Any], _logger: Optional[_EarlyLogger] = None
    ) -> None:
        param_map = dict(key="api_key", host="base_url", timeout="login_timeout")
```

Reply:
Thanks. This is done.